test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8Client.java

```diff
@@ -0,0 +1,636 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.system.framework.kubernetes;
+
+import com.google.gson.reflect.TypeToken;
+import io.kubernetes.client.ApiCallback;
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.ApiException;
+import io.kubernetes.client.Configuration;
+import io.kubernetes.client.PodLogs;
+import io.kubernetes.client.apis.ApiextensionsV1beta1Api;
+import io.kubernetes.client.apis.AppsV1Api;
+import io.kubernetes.client.apis.CoreV1Api;
+import io.kubernetes.client.apis.CustomObjectsApi;
+import io.kubernetes.client.apis.RbacAuthorizationV1beta1Api;
+import io.kubernetes.client.models.V1ContainerState;
+import io.kubernetes.client.models.V1ContainerStateTerminated;
+import io.kubernetes.client.models.V1ContainerStatus;
+import io.kubernetes.client.models.V1DeleteOptions;
+import io.kubernetes.client.models.V1Deployment;
+import io.kubernetes.client.models.V1Namespace;
+import io.kubernetes.client.models.V1ObjectMeta;
+import io.kubernetes.client.models.V1Pod;
+import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.models.V1PodStatus;
+import io.kubernetes.client.models.V1beta1ClusterRole;
```

Comment:
What's the deal with these classes that are marked beta? Are these beta features in Kubernetes, in this client or what?

Reply:
They are beta features in Kubernetes client.